charts/lakefs/templates/arktika-deployment.yaml

```diff
@@ -0,0 +1,112 @@
+{{- if .Values.mds.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "arktika.fullname" . }}
+  labels:
+    {{- include "arktika.labels" . | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
Replica exactly 1 is a requirement?
If yes, please add comment just in case people try to tweak it in run-time cuz they will think "oh the amazing people at treeverse forgot to make this a variable"